server/routes/daily-attendance-summary/handlers/attendance.ts

```diff
@@ -0,0 +1,210 @@
+import { Request, Response } from 'express'
+import { convertToArray, formatDate, toDate } from '../../../utils/utils'
+import ActivitiesService from '../../../services/activitiesService'
+import { AttendanceSummaryFilters, FilterItem } from '../../../@types/activities'
+import PrisonService from '../../../services/prisonService'
+import AttendanceStatus from '../../../enum/attendanceStatus'
+import AttendanceReason from '../../../enum/attendanceReason'
+import { AllAttendance } from '../../../@types/activitiesAPI/types'
+
+export default class DailyAttendanceRoutes {
+  constructor(private readonly activitiesService: ActivitiesService, private readonly prisonService: PrisonService) {}
+
+  GET = async (req: Request, res: Response): Promise<void> => {
+    const { user } = res.locals
+    const activityDate = req.query.date ? toDate(req.query.date as string) : undefined
+    if (activityDate === undefined) {
+      return res.redirect('select-period')
+    }
+    let { status } = req.query
+    if (status === undefined) {
+      status = ''
+    }
```

Comment:
Just an fyi, but you can do this conditional assignment in a single line with this:

```suggestion
     status ??= ''
```